libeos-updater-util/tests/config.c

```diff
@@ -240,6 +240,28 @@ test_config_file_nonexistent (Fixture       *fixture,
   g_assert_cmpuint (loaded_file, ==, 1);
 }
 
+/* Test that if none of the files exist, but the GResource does, we successfully
+ * use that.. */
+static void
+test_config_file_resource_only (Fixture       *fixture,
+                                gconstpointer  user_data G_GNUC_UNUSED)
+{
+  g_autoptr(GError) error = NULL;
+  g_autoptr(EuuConfigFile) config = NULL;
+  const gchar * const paths[] =
+    {
+      fixture->key_file_nonexistent_path,
+      NULL
+    };
+  guint loaded_file;
+
+  config = euu_config_file_new (paths, fixture->default_resource, fixture->default_path);
+
+  loaded_file = euu_config_file_get_uint (config, "Test", "File", 0, G_MAXUINT, &error);
```

Comment:
If `paths` is moved above `config` these can probably all be declared and initialized on the same line

Reply:
Meh, not worth it.